contracts/StrategyProviderWallet.sol

```diff
@@ -98,7 +105,11 @@ contract StrategyProviderWallet is IStrategyProviderWallet {
         address pool = IUniswapV3Factory(uniswapAddressHolder.uniswapV3FactoryAddress()).getPool(token0, token1, fee);
         require(pool != address(0), "SPWAP0");
 
-        checkPerformanceFeeRatio(performanceFeeRatio);
+        ///@dev check performanceFeeRatio only if the official account is not the owner
+        if (owner != registry.officialAccount()) {
+            checkPerformanceFeeRatio(performanceFeeRatio);
+        }
```

Comment:
add protection for official account config error?

Reply:
what kind of config error?

Reply:
such as rate greater than 100%.